contracts/src/v0.8/shared/test/helpers/ChainReaderTester.sol

```diff
@@ -26,11 +26,11 @@ struct InnerTestStruct {
 contract ChainReaderTester {
   event Triggered(
     int32 indexed field,
-    string differentField,
     uint8 oracleId,
     uint8[32] oracleIds,
     address Account,
     address[] Accounts,
+    string differentField,
     int192 bigField,
```

Comment:
I don't think it is necessary to move the `string` field, rather we should be able to just use index 1 for `oracleId` here and the LogPoller will read the 2nd 32 byte data word. But maybe that will break things that are outside the scope for this ticket that I will handle in my PR.

Reply:
The indexes from the cfg are not used in this way, which is why I just removed them. Indexes are calculated internally on CR init.

Reply:
I see, currently the loop in `getEventTypes` skips the rest of the fields if it has seen a dynamic type.